build.sbt

```diff
@@ -100,9 +100,12 @@ lazy val root = Project(id = "pekko", base = file("."))
       akkaScalaNightly,
       docs,
       serialversionRemoverPlugin))
-  .settings(Compile / headerCreate / unmanagedSources := (baseDirectory.value / "project").**("*.scala").get)
+  .settings(
+    Compile / headerCreate / unmanagedSources := (baseDirectory.value / "project").**("*.scala").get,
+    sourceDistName := "pekko")
```

Comment:
incubator-pekko is more correct for the source distribution

Reply:
Fixed